website/content/docs/connect/config-entries/exported-services.mdx

```diff
@@ -298,7 +298,7 @@ spec:
 
 <Tab heading="Consul Enterprise (Peers)">
 
-The following example configures Consul to export the `payments` and `refunds` services from the `billing` namespace of the `finance` admin partition to the `web-shop` peer.
+The following example configures Consul to export the `payments` and `refunds` services from the `billing` namespace of the `finance` admin partition to that partition's `web-shop` peer.
```

Comment:
I'll write out what's fully accurate. Can you help me figure out how to most clearly convey that?

```suggestion
The following example configures Consul to export the `payments` and `refunds` services from the `billing` namespace of the `finance` admin partition to the `finance` partition's `web-shop` peer.
```

Reply:
Namespaces belong to partitions.

A "peer" is a relative term. It means that a partition in this datacenter has a relationship with a partition in another datacenter. Just saying "the `web-shop` peer" can be misleading, because it's not clear which entity has that peering relationship.

`web-shop` is the name that the `finance` partition uses to refer to some remote partition.

Reply:
I'm fine with either version, but I have a slight preference for my suggestion. This is because 1) it avoids using the possessive form for the inanimate "`finance` partition" and 2) I think it's clearer to state that the `billing` namespace is _in_ the `finance` partition than being _of_ the `finance` partition.

Reply:
I agree with the change to "in the `finance` partition".

I'm not sure the wording "the `web-shop` peer, which is also in the `finance` partition" is suitable. The thing you are peering with doesn't exist in the `finance` partition. The peering _relationship_ exists in the `finance` partition, but the peer itself does not.